src/test/java/org/fcrepo/client/integration/FcrepoAuthenticationIT.java

```diff
@@ -29,109 +35,207 @@
+            final String content = IOUtils.toString(response.getBody(), "UTF-8");
+            final int status = response.getStatusCode();
+            assertEquals("Didn't get a CREATED response! Got content:\n" + content,
+                    CREATED.getStatusCode(), status);
+        }
     }
 
     @Test
     public void testUnAuthUserCannotPost() throws Exception {
-        final InputStream body = new ByteArrayInputStream(rdfTtl.getBytes());
-        final FcrepoResponse response = client.post(new URI(serverAddress))
-                .body(body, TEXT_TURTLE)
-                .perform();
-        final String content = IOUtils.toString(response.getBody(), "UTF-8");
-        final int status = response.getStatusCode();
-        assertEquals("Unauthenticated user should be forbidden! Got content:\n" + content,
-                FORBIDDEN.getStatusCode(), status);
+        try (final InputStream body = new ByteArrayInputStream(rdfTtl.getBytes());
+                final FcrepoResponse response = unauthClient.post(new URI(SERVER_ADDRESS))
+                        .body(body, TEXT_TURTLE)
+                        .perform()) {
+            final String content = IOUtils.toString(response.getBody(), "UTF-8");
+            final int status = response.getStatusCode();
+            assertEquals("Unauthenticated user should be forbidden! Got content:\n" + content,
+                    FORBIDDEN.getStatusCode(), status);
+        }
     }
 
     @Test
     public void testAuthUserCanGet()
             throws Exception {
-        final FcrepoResponse response = authClient.get(new URI(serverAddress)).perform();
-        final int status = response.getStatusCode();
-        assertEquals("Authenticated user can not read root!", OK
-                .getStatusCode(), status);
+        try (final FcrepoResponse response = authClient.get(new URI(SERVER_ADDRESS)).perform()) {
+            final int status = response.getStatusCode();
+            assertEquals("Authenticated user can not read root!", OK
+                    .getStatusCode(), status);
+        }
+
+    }
+
+    @Test
+    public void testAuthUserNoHostCanGet()
+            throws Exception {
+        try (final FcrepoResponse response = authClientNoHost.get(new URI(SERVER_ADDRESS)).perform()) {
+            final int status = response.getStatusCode();
+            assertEquals("Authenticated user can not read root!", OK
+                    .getStatusCode(), status);
+        }
     }
 
     @Ignore("Pending alignment with WebAC in FCREPO-2952")
     @Test
     public void testUnAuthUserCannotGet()
             throws Exception {
-        final FcrepoResponse response = client.get(new URI(serverAddress)).perform();
-        final int status = response.getStatusCode();
-        assertEquals("Unauthenticated user should be forbidden!", FORBIDDEN
-                .getStatusCode(), status);
+        try (final FcrepoResponse response = unauthClient.get(new URI(SERVER_ADDRESS)).perform()) {
```

Comment:
It looks like this method is still ignored, and the ticket it references is "Won't Fix". I wonder if we should re-enable it or delete it. It apparently does not pass